src/main/java/nl/nn/testtool/web/FrontendServlet.java

```diff
@@ -0,0 +1,110 @@
+package nl.nn.testtool.web;
+
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.MediaType;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.invoke.MethodHandles;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+public class FrontendServlet extends HttpServlet {
+
+	private static final long serialVersionUID = 123L;
+	private Map<String, MediaType> mediaTypeMap;
+	private final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+	@Override
+	public void init() throws ServletException {
+		super.init();
+		System.err.println("INITIALIZING FRONTEND SERVER!!!!!!");
```

Comment:
I think we should remove this before merging to master?

Reply:
This has been removed